metrics/storage/blogbench.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'blogbench', and extracts the 'scores' for reads
+# and writes
+# Note - the scores are *not* normalised for the number of iterations run,
+# they are total scores for all iterations (this is the blogbench default output)
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+TEST_ARGS="$@"
+
+TEST_NAME="blogbench"
+IMAGE="local-blogbench"
+DOCKERFILE="${SCRIPT_PATH}/Dockerfile"
+
+# Number of iterations for blogbench to run - note, results are not
+# scaled to iterations - more iterations results in bigger results
+ITERATIONS="${ITERATIONS:-30}"
+
+# Directory to run the test on
+TESTDIR="${TESTDIR:-/tmp}"
+CMD="blogbench -i ${ITERATIONS} -d ${TESTDIR}"
+
+# Check if we have our local image installed, and if not, install it
+# from the dockerfile
+function local_check_images() {
+	local exists=$(docker image inspect $IMAGE > /dev/null; echo $?)
```

Comment:
As per other note - place consider if we can make this a little smarter wrt versioning.
And, if you do, also maybe we should move this out to the generic lib funcs for use by other tests - I predict we will use this same idea in the network tests for instance ;-)